src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-list/rgw-bucket-list.component.html

```diff
@@ -22,3 +22,24 @@
                          [selection]="expandedRow">
   </cd-rgw-bucket-details>
 </cd-table>
+
+<ng-template #bucketSizeTpl
+             let-row="row">
+  <cd-usage-bar *ngIf="row.size_usage; else noSizeQuota"
+                [total]="row.bucket_quota.max_size"
+                [used]="row.bucket_size">
+  </cd-usage-bar>
+
+  <ng-template #noSizeQuota>No Limit</ng-template>
+</ng-template>
+
+<ng-template #bucketObjectTpl
+             let-row="row">
+  <cd-usage-bar *ngIf="row.object_usage; else noObjectQuota"
```

Comment:
```suggestion
  <cd-usage-bar *ngIf="row.bucket_quota.enabled && row.bucket_quota.max_objects; else noObjectQuota"
```

Reply:
Alfonso there a change in the component ts you might have missed - `object_usage` and `size_usage` only exist if `max_size` or `max_objects` is greater 0. Which means the ng-if would work perfectly like it was.

Reply:
> @alfonsomthd there is a change in the component ts you might have missed - `object_usage` and `size_usage` only exist if `max_size` or `max_objects` is greater 0. Which means the ng-if would work perfectly like it was.

@Devp00l Regarding max_size, OK but ```row.bucket_quota.enabled``` has to be checked at some point (.ts or .html) because you can set quota and the disable it e.g.:
```
radosgw-admin quota set --quota-scope=bucket --uid=testid --max-objects=10 --max-size=1M
radosgw-admin quota enable --quota-scope=bucket --uid=testid --bucket test

radosgw-admin quota disable --quota-scope=bucket --uid=testid --bucket test
```
So you can get ```size_usage/max_size > 0``` but ```enabled: false```